src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -31,14 +31,26 @@
         "SSID", "StudentIdentifier"
     );
 
+    private static final Map<String, String> relationshipNames = ImmutableMap.of(
```

Comment:
Grr - that's annoying.  What about swapping out the `ImmutableMap` to use Java 8's `UnmodifiableMap`?  I don't think `UnmodifiableMap` is perfect (because according to the docs its a "view of a modifiable map that could change"), but if the original map is private and `final`, I think you'd be okay.

Or, maybe since the map is private and final, just use a `HashMap`?  The only thing that has access to this map is the methods in this class, so the odds of it getting modified are pretty slim.

Or, you could just leave it the way it is.  This isn't really a deal-breaker for me; just a little odd that there are two maps that do the same thing.